frontend/packages/webterminal-plugin/integration-tests/support/step-definitions/web-terminal/web-teminal-basic.ts

```diff
@@ -23,3 +25,6 @@ Then(
     webTerminalPage.verifyRestartTerminalButton();
   },
 );
+Given('user with basic rights has installed Web Terminal operator', () => {
+  verifyAndInstallWebTerminalOperator();
+});
```

Comment:
nit: If this verifies a state it should be a `Then(` function, or?

But this new function isn't used here. Do we really need it?

Reply:
Well, you are right we've already had this step implementation here: https://github.com/openshift/console/blob/ff3493dbc21b0e382f01c3fc9abd73a169132c18/frontend/packages/dev-console/integration-tests/support/step-definitions/common/operators.ts#L35
But when I ran the test, I had an error like this :
![Selection_001](https://github.com/openshift/console/assets/1640058/60bd129e-a08d-41e0-9cca-c82c6c0d0788)
After investigating the problem, I found the article: 
https://stackoverflow.com/questions/59857488/error-step-implementation-missing-for-step-definition-when-trying-to-run-a-s
And seems the main problem: `Folder name and Feature file name should be the same`. Maybe it can be fixed more elegantly, but I do not know how. Unfortunately, my experience is not huge with Cypress/cucumber